### PR TITLE
make cmd short consistency

### PR DESCRIPTION
**Pull request by allencloud**
make cmd short consistency
Signed-off-by: allencloud allen.sun@daocloud.io

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

Have you looked at all cmds?

**Comment by allencloud** [comment]
Yes. One more time, and change short of volume inspect.

**Comment by duglin** [comment]
thanks, stll LGTM

**Comment by vdemeester** [comment]
LGTM 🐱

**Comment by allencloud** [comment]
@duglin @vdemeester 
PTAL
Changed a lot in the docs

**Comment by vdemeester** [comment]
@allencloud docs change are going to duplicate with that PR #24097 on the API changes though but no biggies, still LGTM :angel:

File: docs/reference/api/docker_remote_api_v1.24.md [hunk 1]
```diff
@@ -3460,7 +3460,7 @@ Return low-level information on the node `id`
 
 **Example request**:
 
-      GET /node/24ifsmvkjbyhk HTTP/1.1
+      GET /nodes/24ifsmvkjbyhk HTTP/1.1
```
Comment (duglin) [inline docs/reference/api/docker_remote_api_v1.24.md]: nice catch

**Comment by duglin** [comment]
still LGTM

**Pull request merged**